Seungwoon/extra/chp6_ctrlZ.py

```diff
@@ -0,0 +1,23 @@
+# 차례대로 더하다가 Z 나오면 해당 숫자 빼기 -> 완전 stack 그 자체
+# 음수인 경우 - 부호까지 합쳐서 문자열 길이가 길어진다. -> list 변환 필요
+def solution1(s):  #stack을 사용하지 않은 풀이
+    s = list(s.split()) # 각 원소 list 변환
+    answer = 0
+    for i, val in enumerate(s): # 인덱스 연산이 필요하면 걍 enumerate 쓰자
```

Comment:
불필요한 list 캐스팅 없이 아래와 같이 코딩해도 괜찮을 것 같습니다
```suggestion
    answer = 0
    for i, val in enumerate(s.split()): # 인덱스 연산이 필요하면 걍 enumerate 쓰자
```